app/javascript/dashboard/routes/dashboard/contacts/components/ReminderItem.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div class="item-wrap">
+    <div class="radio-button">
+      <input :checked="isCompleted" type="radio" />
+    </div>
+    <div class="wrap">
+      <div class="text">
+        {{ text }}
+      </div>
+      <div class="footer">
+        <div class="meta">
+          <div class="date-wrap">
+            <i class="icon ion-android-calendar" />
+            <span class="date">
+              {{ date }}
+            </span>
+          </div>
+          <div class="label-wrap">
+            <div class="label-text">
+              {{ label }}
+            </div>
+          </div>
+        </div>
+        <div class="actions">
+          <woot-button
+            variant="clear"
+            size="small"
+            icon="ion-compose"
+            color-scheme="secondary"
+            class-names="button--emoji"
+            class="button-wrap"
+            @click="onEdit"
+          />
+          <woot-button
+            variant="clear"
+            size="small"
+            icon="ion-trash-b"
+            color-scheme="secondary"
+            class-names="button--emoji"
+            class="button-wrap"
+            @click="onDelete"
+          />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    id: {
+      type: Number,
+      default: 0,
+    },
+    text: {
+      type: String,
+      default: '',
+    },
+    isCompleted: {
+      type: Boolean,
+      default: false,
+    },
+    date: {
+      type: Number,
+      default: 0,
+    },
+    label: {
+      type: String,
+      default: '',
+    },
+  },
+
+  methods: {
```

Comment:
Need to emit an event when user clicks radio button to mark a task as completed.